### PR TITLE
Update for hapi log response

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -10,10 +10,6 @@ const register = async function(server, options) {
     if (!event.data) {
       return;
     }
-    if (options.clientErrorsToWarnings && event.tags.indexOf('error') !== -1 && event.tags.indexOf('client') !== -1) {
-      event.tags[event.tags.indexOf('error')] = 'warning';
-    }
-    log(event.tags, event.data, { addErrorTagToErrors: !options.clientErrorsToWarnings });
```
Comment (jgallen23) [inline index.js]: umm, you removed the whole point of the library...

File: index.js [hunk 2]
```diff
@@ -10,10 +10,6 @@ const register = async function(server, options) {
...
   });
   if (options.requests === true) {
```
Comment (jgallen23) [inline index.js]: we should remove this to log-response as an option to log all requests

File: index.js (hunk 1 above)
Comment (jgallen23) [reply index.js]: this scares me a lot - means that the tests weren't actually testing anything.

Comment (orthagonal) [reply index.js]: Have updated the tests to ensure we are actually passing the info to logr.

**Pull request merged**